ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -356,47 +380,34 @@ open class TwoLineTitleView: UIView, TokenizedControlInternal {
         }
     }
 
-    private func setupButton(_ button: UIButton, label: UILabel, trailingImageView: UIImageView, text: String?, interactive: Bool, accessoryType: AccessoryType) {
-        button.isUserInteractionEnabled = interactive
-        button.accessibilityLabel = text
+    private func setupTitleLine(_ container: UIStackView, label: UILabel, trailingImageView: UIImageView, imageSize: GlobalTokens.IconSizeToken, text: String?, interactive: Bool, accessoryType: AccessoryType) {
```

Comment:
Considering we only want two out of nine possible `GlobalTokens.IconSizeToken`s, I don't think it's the best choice to use `IconSizeToken` as the type you're passing in here. Maybe just a `Bool` for if it's title or subtitle would make more sense.

Reply:
Using a `Bool` saves a few kilobytes of binary size. Works for me.